migration_scripts/README.md

```diff
@@ -0,0 +1,145 @@
+- [Prerequisites](#prerequisites)
+- [Migration Scripts](#migration-scripts)
+- [Backup Content and Config](#backup-content-and-config)
+- [Create Mappings](#create-mappings)
+- [Move files](#move-files)
+- [Create redirects](#create-redirects)
+- [Sidebar Unification](#sidebar-unification)
+- [Check That All Pages Are Linked to in the Sidebar](#check-that-all-pages-are-linked-to-in-the-sidebar)
+- [Position Files and Folders - Ready For Testing](#position-files-and-folders---ready-for-testing)
+- [Testing Changes Locally](#testing-changes-locally)
+- [Push Changes](#push-changes)
+- [Clean Up](#clean-up)
+
+# Prerequisites
+
+```bash
+pip3 install openpyxl
+```
+
+# Migration Scripts
+
+Fork the developer repository, clone your fork, create a new branch, set the remote:
+
+```bash
+cd ~
+git clone git@github.com:yourusername/developer.git
+cd developer
+git checkout -b my_new_branch
+git remote add upstream https://github.com/fermyon/developer
+```
+
+Change into the migration_scripts directory and then run the following scripts:
+
+```bash
+cd migration_scripts
+```
+
+# Backup Content and Config
+
+Run the following shell script to make it easy to reset (iterate while making decisions about the migration)
+
+```bash
+./backup_content.sh
+```
+
+# Create Mappings
+
+Run the create_url_vs_markdown_file_vs_toc_label_mapping.py script. It will generate a spreadsheet called mapping_information.xlsx which will contain all of the current url paths from the existing spin v2 sidebar template and the cloud sidebar template:
+
+```bash
+python3 create_url_vs_markdown_file_vs_toc_label_mapping.py
+```
+
+Open the mapping_information.xlsx spreadsheet and fill in the:
+- unified_url_path (the new url where the resource will be found online i.e. `/spin/v2/quickstart` vs `/latest/quickstart` etc.)
```

Comment:
Just a quick note that I believe we had potentially opted on `/docs` over `/latest` for the v3 paths?

Reply:
100% perfect - thanks so much.
Updating this all in the next push.